MAVProxy/modules/mavproxy_chat/chat_openai.py

```diff
@@ -327,6 +379,49 @@ def get_vehicle_type(self):
             "vehicle_type": vehicle_type_str
         }
 
+    # get mode mapping
```

Comment:
Comment could be better... I exepct the "as a json string" is implied in here somehow.